src/System.Text.JsonLab.Serialization/System/Text/Json/Serialization/TempExtensionMethods.cs

```diff
@@ -1,43 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-namespace System.Text.Json.Serialization
-{
-    // Temporary extension methods until we move to corefx
-    internal static class TempExtensionMethods
-    {
-        // TODO: Replace with existing static helper in corefx
-        // System/Text/Json/Reader/JsonReaderHelper.Unescaping.cs#L42
-        public static string TranscodeHelper(ReadOnlySpan<byte> utf8Unescaped)
```

Comment:
Why remove this?

Reply:
See previous - I added it aback but renamed the class to match corefx.